src/sagas/api.js

```diff
@@ -68,8 +72,17 @@ export function* apiPost (path, options) {
     data,
     {
       headers,
+      // Always return true so error status 4xx promises aren't rejected.
+      axiosValidateStatus,
```

Comment:
This doesn't do what you think it's doing. Before the force push, the config object was of the form:

```javascript
{
  headers: {
    'Content-Type': 'application/json',
    ...
  },
  validateStatus: () => true,
}
```

The code you have now which leverages the es6 shorthand results in this format:
```javascript
{
  headers: {
    'Content-Type': 'application/json',
    ...
  },
  axiosValidateStatus: {
    validateStatus: () => true,
  }
}
```

I would wager that this no longer solves the problem the PR is setting out to fix since axios knows nothing of a `axioValidateStatus` configuration variable.